CTNotificationContent/Templates/ZeroBezel/Controller/CTZeroBezelController.swift

```diff
@@ -166,6 +191,35 @@ import UIKit
         return deeplinkURL
     }
     
+    @objc public override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
+            super.traitCollectionDidChange(previousTraitCollection)
+            
+            // Only handle trait changes on iOS 16 and below
+            // iOS 17+ uses registerForTraitChanges
+            if #available(iOSApplicationExtension 17.0, *) {
+                // Do nothing, handled by registerForTraitChanges
+            } else if #available(iOSApplicationExtension 12.0, *) {
+                if traitCollection.userInterfaceStyle != previousTraitCollection?.userInterfaceStyle {
+                    updateInterfaceColors()
+                }
+            }
+    }
+    
+    func updateInterfaceColors() {
+        // Check if device is in dark mode (iOS 12+)
+        let isDarkMode: Bool
+        
+        if #available(iOSApplicationExtension 12.0, *) {
```

Comment:
Create a common util method which returns bool and use at every place since this code is repeated everywhere
if #available(iOSApplicationExtension 12.0, *) {
            isDarkMode = traitCollection.userInterfaceStyle == .dark
        } else {
            // For iOS versions before 12.0,using light mode colors since dark mode wasn't officially supported
            isDarkMode = false
        }